roles/confluent.kafka_connect_replicator/templates/override.conf.j2

```diff
@@ -0,0 +1,15 @@
+[Service]
```

Comment:
Please incorporate the recent "Unit file customizations" logic that was just merged into the 6.1.x branch